lib/index.js

```diff
@@ -31,7 +31,7 @@ function logEvent(ctx, data, request) {
   } else if (ctx.includeData && data.data !== undefined) {
     if (ctx.mergeData) {
       try {
-        lodash.assign(obj, data.data);
+        obj = lodash.assign(data.data, obj);
```

Comment:
We can't modify data people send into a logging library, it's going to screw up other plugins.